pipeline/pipeline-scw.yml

```diff
@@ -0,0 +1,386 @@
+# YAML anchors
```

Comment:
Do we need an entier new pipeline or could we rely on our latest way to give all terraform config as variable to keep only one pipeline ?

Reply:
we can't, terraform backend needs extra parameters to work with scaleway